src/test/java/stringaddcalculator/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,79 @@
+package stringaddcalculator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringAddCalculatorTest {
+
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
+
+        result = StringAddCalculator.splitAndSum("");
+        assertThat(result).isEqualTo(0);
+    }
+
+    @Test
+    public void splitAndSum_숫자하나() throws Exception {
```

Comment:
```suggestion
    void splitAndSum_숫자하나() {
```
1. `throws Exception`은 제거되어도 될 것 같아요.
2. junit5부터는 메소드의 pulibc이 제거되어도 됩니다 (https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods)